.github/workflows/docbuild.yml

```diff
@@ -0,0 +1,16 @@
+name: 'Release Website Publish'
+on:
+  release:
+    types: [published]
+permissions:
+  contents: write
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: 3.x
+      - run: pip install mkdocs-material
+      - run: mkdocs gh-deploy --force
```

Comment:
Can we  run `make push-mkdocs` here instead? The reason is we generate API documentation as part of the push, i.e. https://aws-quickstart.github.io/cdk-eks-blueprints/api/. 
If we just perform gh-deploy it won't be there. 
An alternative is to run 
```
npx typedoc --out docs/api lib/index.ts
mkdocs gh-deploy
```

Reply:
makes a lot of sense. I didnt think about our Make process. fixed it.